python/examples/experimental/Condition_Validator_UDF.ipynb

```diff
@@ -0,0 +1,225 @@
+    "\n",
+    "## Example\n",
+    "\n",
+    "Let's say you are logging a numerical column `col1`, and you want to trigger an action whenever the evaluated row value for this column is greater than 4. To do so, we'll define two functions: an action and a condition. We will then decorate the condition function with the `condition_validator` decorator, and pass the action function as an argument."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Note: you may need to restart the kernel to use updated packages.\n",
+    "%pip install whylogs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "No session found. Call whylogs.init() to initialize a session and authenticate. See https://docs.whylabs.ai/docs/whylabs-whylogs-init for more information.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Validator: less_than_four\n",
+      "    Condition name less_than_four failed for value 7\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<whylogs.core.view.dataset_profile_view.DatasetProfileView at 0x7fe2a606a520>"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "from typing import Any\n",
+    "from whylogs.experimental.core.validators import condition_validator\n",
+    "from whylogs.experimental.core.udf_schema import udf_schema\n",
+    "import whylogs as why\n",
+    "\n",
+    "data = pd.DataFrame({\"col1\": [1, 3, 7]})\n",
+    "\n",
+    "def do_something_important(validator_name, condition_name: str, value: Any, column_id=None):\n",
+    "    print(\"Validator: {}\\n    Condition name {} failed for value {}\".format(validator_name, condition_name, value))\n",
+    "    return\n",
+    "\n",
+    "@condition_validator([\"col1\"], condition_name=\"less_than_four\", actions=[do_something_important])\n",
+    "def lt_4(x):\n",
+    "    return x < 4\n",
+    "\n",
+    "schema = udf_schema()\n",
+    "why.log(data, schema=schema).view()\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can see that the action was triggered once for the value 7.\n",
+    "\n",
+    "Condition Validators are compatible with Dataset UDFs. Through Dataset UDFs, you can create new columns based on the values of other columns. In this example, we will create a new column `add5` that is equal to `col1` + 5. We will then use assing a condition validator to the newly created column:"
```

Comment:
Typo. Not sure if we were aiming for "We will then assign a condition validator..." or "We will then apply a condition validator..." or ?